src/server/api/routers/book.ts

```diff
@@ -1,27 +1,203 @@
 import { z } from "zod";
+import { createTRPCRouter, protectedProcedure } from "src/server/api/trpc";
 
-import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+export interface Section {
```

Comment:
think you shouldn't need these. can be inferred by either `routerOutputs` or from Prisma.getPayload, probably the former is better.